src/main/java/com/bigcommerce/BigcommerceSdk.java

```diff
@@ -17,47 +17,13 @@
 import javax.ws.rs.core.Response;
 import javax.ws.rs.core.Response.Status;
 
+import com.bigcommerce.catalog.models.*;
```

Comment:
Please refrain from using wildcard imports and import the specific classes you need. It may need all of the class from that package now, but as we add more models/packages this may change.

See. 3.3.1 of our style guide:
https://channelape.github.io/styleguide/java/javaguide.html